src/Behat/MinkExtension/Context/RawMinkContext.php

```diff
@@ -159,7 +159,7 @@ public function saveScreenshot($filename = null, $filepath = null)
         // Under Cygwin, uniqid with more_entropy must be set to true.
         // No effect in other environments.
         $filename = $filename ?: sprintf('%s_%s_%s.%s', $this->getMinkParameter('browser_name'), date('c'), uniqid('', true), 'png');
-        $filepath = $filepath ? $filepath : (ini_get('upload_tmp_dir') ? ini_get('upload_tmp_dir') : sys_get_temp_dir());
+        $filepath = $filepath ?: (ini_get('upload_tmp_dir') ? ini_get('upload_tmp_dir') : sys_get_temp_dir());
```

Comment:
2 ternary operators on same line is bad. You'd better use regular block IFs instead.